### ISSUE TITLE
onLoadSuccess called infinitely when file={someBlob}

### ISSUE THREAD

**Issue by lorencole** [wojtekmaj/react-pdf#54]
I'm trying to use a Blob for the pdf value, but instead of calling onLoadSuccess once it calls it over and over ad nauseum. 

	import React, { Component, PropTypes } from 'react';
	import { Page } from 'react-pdf';
	import { Document } from 'react-pdf/build/entry.webpack';	// use entry.webpack to enable PDF.js worker
	import b64toBlob from 'b64-to-blob';

 	export class PdfThumbnails extends Component {
		static propTypes = {
			b64data: PropTypes.string.isRequired,
		}

		constructor(props) {
			super(props);
			this.state = {
				numPages: 0,
			}
		}

		onDocumentLoad(x) {
			console.log(x);
	                this.setState({ ...this.state, numPages:x.numPages });
        	}

		render() {
			const { b64data } = this.props;
			let fileBlob = b64toBlob(b64data);

			return (
				<div>
					<Document
						// file="/img/Catalist_identity.pdf"
						file={ fileBlob }
						onLoadSuccess={ this.onDocumentLoad.bind(this) }
					>
						<Page
							pageNumber={ 1 }
							scale={ 0.2 }
						/>
					</Document>
				</div>
			);
		}
	}
	export default PdfThumbnails;

Using a static file resource and the same onLoadSuccess method the component works great.

What I'm really doing here is using a base64 string as source data for <Document />, so an example of that would be very welcome.  The one in https://github.com/wojtekmaj/react-pdf/issues/42 doesn't work for my flavor of ES..

Thanks!

**Comment by saman** [comment]
Try this one for base64
```javascript
file={`data:application/pdf;base64,${b64data}`}
```

I actually wrote this function to work also with base64 for and file URL:
```javascript
  static decodeBase64(str) {
    if (str !== null && str !== undefined && str.length) {
      try {
        atob(str);
        return `data:application/pdf;base64,${str}`;
      } catch (e) {
        return str;
      }
    }
    return '';
  }
```

**Comment by wojtekmaj** [comment]
Hey guys, thank you for this report! Version 2.1.0 will have refactored loading function and I'll make sure this issue is resolved. It should be out in the next 24 hours!

**Comment by wojtekmaj** [comment]
Fixed in 2.1.0.
